src/tests/containerizer/cgroups2_tests.cpp

```diff
@@ -123,6 +123,43 @@ TEST_F(Cgroups2Test, CGROUPS2_Path)
   EXPECT_EQ("/sys/fs/cgroup/foo/bar", cgroups2::path("foo/bar"));
 }
 
+
+TEST_F(Cgroups2Test, CGROUPS_Path)
```

Comment:
This test is about cgroups2::processes and process assignment, not path?

If this is fixed to be s/CGROUPS/CGROUPS2/ then this clashes with the existing CGROUPS2_Path test?

There's already ROOT_CGROUPS2_AssignProcessToCgroup which seems to be about the same as this test, minus cgroups2::processes?

Reply:
Removed the `ROOT_CGROUPS2_AssignProcessToCgroup` test and updated the test name.